Spanner/src/Connection/Grpc.php

```diff
@@ -1111,6 +1111,8 @@ public function beginTransaction(array $args)
             $pdml = new PartitionedDml();
             $options->setPartitionedDml($pdml);
             $args = $this->addLarHeader($args, $this->larEnabled);
+        } elseif (isset($transactionOptions['isolationLevel'])) {
```

Comment:
This shouldn't be an `elseif` because it's a separate field, and is not part of the `oneof mode` (see https://github.com/googleapis/googleapis/blob/4657bfafc269c507073f426376c241e07cdfe2f7/google/spanner/v1/transaction.proto#L509-L530) 

It should be more like `excludeTxnFromChangeStreams` below, which is a separate field